pygfx/renderers/wgpu/_shader.py

```diff
@@ -1,15 +1,24 @@
+from pkg_resources import resource_filename
+
 from ...resources import Buffer, Texture
 from ._utils import to_vertex_format, to_texture_format, GfxSampler, GfxTextureView
 from ._shaderbase import BaseShader
 
 
+def load_shader(shader_name):
+    filename = resource_filename("pygfx.data_files", f"shaders/{shader_name}")
```

Comment:
do you need to change anything about setup.py (and perhaps pyinstaller) to make sure the files in that folder are packaged?

Reply:
I will add a test for this to make sure.